legal-api/migrations/versions/0c3eb026efa0_alter_business_state_enum.py

```diff
@@ -0,0 +1,37 @@
+"""alter_business_state_enum
+
+Revision ID: 0c3eb026efa0
+Revises: 1ec937c2466e
+Create Date: 2025-06-04 08:48:56.728321
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = '0c3eb026efa0'
+down_revision = '1ec937c2466e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    bind = op.get_bind()
+    inspector = sa.inspect(bind)
+    op.execute("ALTER TYPE state RENAME TO state_old")
+
+    state_new = postgresql.ENUM('ACTIVE', 'HISTORICAL', name='state')
+    state_new.create(bind, checkfirst=True)
+
+    op.execute("ALTER TABLE businesses ALTER COLUMN state TYPE state USING state::text::state")
+    op.execute("ALTER TABLE businesses_version ALTER COLUMN state TYPE state USING state::text::state")
+    if 'businesses_bak' in inspector.get_table_names():
```

Comment:
Can't remember the background of `businesses_bak` table (previous legal name change work?)...seems it was added manually only in dev. But it's causing some issues for alembic script if we don't alter this table and don't make this operation optional.